src/useQueryParam.ts

```diff
@@ -9,6 +10,7 @@ import {
 import { QueryParamContext } from './QueryParamProvider';
 import { updateUrlQuery } from './updateUrlQuery';
 import { UrlUpdateType } from './types';
+import { type } from 'os';
```

Comment:
Why is this imported?

Reply:
Probably an IntelliSense import, got rid of that.